pkg/github/projects/projects.go

```diff
@@ -120,8 +138,37 @@ func GetAllProjects(ctx context.Context, client Client, opts models.ListProjects
 	return projects, nil
 }
 
+func getAllProjectsByUser(ctx context.Context, client models.Client, opts models.ProjectOptions) (Projects, error) {
+	var (
+		variables = map[string]interface{}{
+			"cursor": (*githubv4.String)(nil),
+			"login":  githubv4.String(opts.User),
+		}
+
+		projects = Projects{}
+	)
+
+	for {
```

Comment:
We should make an issue to add a page limit so we dont have endless loops

Reply:
added imit